schemas/config-editor.xsd

```diff
@@ -1154,6 +1155,119 @@ Note: Only sections with forEach support del attribute.
     </xs:unique>
   </xs:element>
 
+
+
+  <xs:element name="list">
+    <xs:annotation>
+      <xs:documentation><![CDATA[
+Adding a list
+-------------
+
+The list can be used to display a list of contact with only organisation name for example.
+This can facilitate building simple form hiding the details of ISO elements. In this case,
+use a contact directory to add new contact makes sense.
+
+This mode can be also more efficient when record contains lots of contacts which in some case
+can create large HTML form which can take time to build, download and render in the browser
+(eg. when having more than 100 contacts in a record). If only `label` are used the form will be faster to render.
+
+eg.
+
+..code-block:: xml
+
+
+        <action type="add"
+                forceLabel="true"
+                addDirective="data-gn-directory-entry-selector"
+                or="pointOfContact"
+                in="/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification">
+          <directiveAttributes
+                  data-template-add-action="false"
+                  data-search-action="true"
+                  data-popup-action="true"
+                  data-template-type="contact"
+                  data-variables="gmd:role/gmd:CI_RoleCode/@codeListValue~{role}"/>
+        </action>
+
+        <list name="Contact point"
+              xpath="/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:pointOfContact[*/gmd:role/*/@codeListValue = 'pointOfContact']"
+              del="."
+              sortBy="gmd:organisationName/gco:CharacterString">
+          <item href="*[1]/@uuid">
+            <label xpath="*/gmd:organisationName/gco:CharacterString/text()"/>
+          </item>
+        </list>
+
+        <list name="Originator"
+              xpath="/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:pointOfContact[*/gmd:role/*/@codeListValue = 'originator']"
+              sortBy="gmd:organisationName/gco:CharacterString">
+          <item>
+            <field xpath="*/gmd:organisationName"/>
+          </item>
+        </list>
+        ]]></xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:choice minOccurs="1">
+        <xs:element name="item">
+          <xs:annotation>
+            <xs:documentation><![CDATA[
+              An item in the list.
+              ]]></xs:documentation>
+            </xs:annotation>
+            <xs:complexType>
+              <xs:sequence>
+                <xs:element ref="label"/>
+                <xs:element ref="text"/>
```

Comment:
What is the difference between `text` and the other values?

Reply:
See https://github.com/geonetwork/core-geonetwork/blob/main/schemas/config-editor.xsd#L1447-L1453 so you can also add text in the list item. It is not used for now but for example, we could think of
```
Org name
<hr/>
Address ...
```